extension.yaml

```diff
@@ -58,43 +58,53 @@ resources:
 # Learn more in the docs: https://firebase.google.com/docs/extensions/alpha/ref-extension-yaml#params-field
 params:
   - param: COLLECTION_PATH
-    label: Collection path
+    label: Collection Path
     description: >
-      Specify the Collection path to monitor for changes.
+      What is the path to the Cloud Firestore collection where the extension should monitor for changes?
     default: ''
     example: store
     validationRegex: "^[^/]+(/[^/]+/[^/]+)*$"
-    validationErrorMessage: Must be a valid Cloud Firestore collection
+    validationErrorMessage: Firestore collection paths must be an odd number of segments separated by slashes, e.g. "path/to/collection".
     required: true
 
   - param: FIELDS
     label: Indexable Fields
     description: >
-      A comma separated list of Cloud Firestore fields to index. Leave empty to index all fields.
+      What document fields should be indexed to provide the best search experience?
+      This can be comma separated list or left blank to index all fields.
+      Refer to [reducing record size](https://www.algolia.com/doc/guides/sending-and-managing-data/prepare-your-data/how-to/reducing-object-size/).
+      For performance reasons, [record size is limited](https://www.algolia.com/doc/guides/sending-and-managing-data/prepare-your-data/in-depth/index-and-records-size-and-usage-limitations/#record-size-limits).
```

Comment:
Would recommend the other changes in the README applied to the description fields here as well.